src/App.js

```diff
@@ -42,6 +43,7 @@ function App() {
               <Route path="/people" component={People} />
               <Route path="/notes" component={Notes} />
               <Route path="/positions" component={Positions} />
+              <Route path="/:personId" component={PersonDetails} />
```

Comment:
I would make it here for example `path="/person/:personId"`.

Reply:
Adjusted as you proposed